### PR TITLE
add 211 hotline notice to EIP overview page

### PR DESCRIPTION
**Pull request by loumoore**

### CODE REVIEW

File: app/views/questions/eip_overview/edit.html.erb [hunk 1]
```diff
@@ -26,7 +26,14 @@
       </li>
     </ul>
 
-    <button class="button button--primary button--wide text--centered spacing-above-60" name="eip_overview_form[eip_only]" value="true" type="submit">
+    <div class="notice--warning text--small">
+      <p class="spacing-below-25 text--bold"><%=t("views.questions.eip_overview.hotline_notice.title") %></p>
+      <p><%=t("views.questions.eip_overview.hotline_notice.what") %></p>
+      <p><%=t("views.questions.eip_overview.hotline_notice.why") %></p>
+      <p class="text--underline"><%=t("views.questions.eip_overview.hotline_notice.number") %></p>
```
Comment (bengolder) [inline app/views/questions/eip_overview/edit.html.erb:33]: I noticed that you underlined this. That would make it look like a link. You could make it into an actual link that allows a user to initiate a phone call.
```html
Call <a href="tel:+18443223639">+1 844 322 3639</a>
```

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -77,6 +77,10 @@ body {
   font-weight: bold;
 }
 
+.text--underline {
+  text-decoration: underline;
+}
+
```
Comment (bengolder) [inline app/assets/stylesheets/application.scss:83]: I see that we have other typographic styles in this file, but ideally, style classes like `.text--underline` should be in `atoms/typography` if possible. Non-blocking, but would be a small step to better CSS organization.

**Review by bengolder — APPROVED** [review]
approved with nonblocking ideas for possible improvements

File: app/views/questions/eip_overview/edit.html.erb (hunk 1 above)
Comment (loumoore) [reply app/views/questions/eip_overview/edit.html.erb:33]: oh that makes much more sense thanks

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (loumoore) [reply app/assets/stylesheets/application.scss:83]: ah yeah just saw the bold one here already..but will move all the typographic stuff to the right place, thanks!

**Pull request merged**